Eatery Blue/UI/EateryScreen/EateryModelController.swift

```diff
@@ -233,6 +233,10 @@ class EateryModelController: EateryViewController {
             if menuCategory.category == "Chef's Table - Sides" {
                 sortedCategories.swapAt(1, i)
             }
+//            if menuCategory.category == "Grill" {
+//                sortedCategories.swapAt(2, i)
+//                print("works")
+//            }
```

Comment:
Uncomment this code to keep it consistent with what you have above. Let's also remove this print statement.